### ISSUE TITLE
add to accessibilitySummary information which cannot be expressed through other metadata

### ISSUE THREAD

**Issue by GeorgeKerscher** [w3c/publ-a11y#96]
Publishers may want to provide information about features which cannot be expressed through the other metadata fields. For example, This title in the Morse Code section contains audio files that demonstrate Morse Code such as what is heard by telegraph.
